### ISSUE TITLE
Remove System Info And Send People to WP's Site Health Instead

### ISSUE THREAD

**Issue by fpcorso** [PopupMaker/Popup-Maker#862]
Right now, our system info is just a text dump into a textarea. In order for an admin to identify issues, they have to scroll through the textarea and know enough about WordPress and Popup Maker to determine if something is an issue.

We should update it to make it clear what is a problem and what isn't. Probably should also set up some documentation to help explain all the parts of the information too.

I like how some plugins call it "System Status" instead of "System Info". Maybe we consider renaming it?

**Comment by fpcorso** [comment]
@danieliser Something I have seen a few other plugins do recently is to remove their "system info" pages and instead add some checks to the core "Site Health" page. 

For example, Monster Insights and WP Cache do this as shown:
![image](https://user-images.githubusercontent.com/9730040/103672450-50ff1280-4f4a-11eb-8b9f-b7546bf035fd.png)

Then, to get system info, we just ask them to copy the system info as shown:
![image](https://user-images.githubusercontent.com/9730040/103672533-6bd18700-4f4a-11eb-9cc1-96b559484ca2.png)

This way, we are hooking into something the admins are already checking and it is much more user-friendly to spot an issue as opposed to having to scroll through our current text.

Thoughts on going with this strategy as opposed to my proposed improvements on this issue?

**Comment by danieliser** [comment]
@fpcorso That second set of approaches looks good. Hook into existing stuff is way smarter.

**Comment by fpcorso** [comment]
@danieliser I reviewed our support info and what is in the Site Health Info that can be saved to clipboard and all of the info we have is already there so there's nothing for us to add.

Plus, all the things we check for are already in the Site Health checks too.

So, we could remove and send people there already. Should we proceed with this plan then? Would there be anything additional we want to add?

It's one less thing we have to maintain, other people are actively working on it, and it's something that shows up in the dashboard for most admins now.
